vizro-core/examples/features/app.py

```diff
@@ -365,19 +367,28 @@
         * RadioItems (**categorical** single option selector only)
         * RangeSlider (**numerical** multi option selector only)
         * Slider (**numerical** single option selector only)
+        * DatePicker(**temporal** multi and single option selector)
 
         """
         ),
         vm.Table(
-            id="table-gapminder", figure=dash_data_table(data_frame=gapminder, page_size=10), title="Gapminder Data"
+            id="table-gapminder",
+            figure=dash_data_table(data_frame=gapminder_with_dt, page_size=10),
+            title="Gapminder Data",
         ),
         vm.Table(id="table-tips", figure=dash_data_table(data_frame=tips, page_size=10), title="Tips Data"),
     ],
     controls=[
         vm.Filter(
             targets=["table-gapminder"],
-            column="year",
-            selector=vm.RangeSlider(title="Range Slider (Gapminder - year)", step=1, marks=None),
+            column="lifeExp",
+            selector=vm.RangeSlider(
+                title="Range Slider (Gapminder - lifeExp)",
+                step=1,
+                marks=None,
+                min=int(gapminder["lifeExp"].min()),
+                max=int(gapminder["lifeExp"].max()) + 1,
```

Comment:
Do we need to set these `min`/`max` explicitly for some reason?

Reply:
I did it that way only so that min and max would be integers (it looks nicer on the screen).

Reply:
I would probably remove that, otherwise the yaml and python version are out of sync, because you can't insert python functions in yaml. 

Could you double-check that the two versions (pydantic + yaml) output the same result?

Reply:
You're right, the yaml example has the different output. So, let's revert these changes 😄.